spidermonkey/include/android/js/Value.h

```diff
@@ -342,10 +342,7 @@ typedef union jsval_layout
 # endif /* JS_PUNBOX64 */
 #endif  /* defined(IS_LITTLE_ENDIAN) */
 
-//this is a workaround for fixing binding-generator errors
```

Comment:
Removes the workaround.
Bindings-generator should handle this correctly.
Don't add workaround here.